payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/DateConfig.kt

```diff
@@ -33,15 +33,15 @@ internal class DateConfig : TextFieldConfig {
             Error.Blank
         } else {
             val newString = convertTo4DigitDate(input)
-            when {
+            return when {
```

Comment:
We don't need this `return` (or the one that existed before).

Reply:
True!